### ISSUE TITLE
Installation fails due to missing dependency: `@modelcontextprotocol/sdk@^0.1.0`

### ISSUE THREAD

**Issue by Starfish-Fujieda** [qpd-v/mcp-guide#1]
Hello,

I'm encountering an issue when trying to install the `mcp-guide` package. The installation fails with an error indicating a missing dependency: `@modelcontextprotocol/sdk@^0.1.0`. It appears that this specific version of the `@modelcontextprotocol/sdk` package is not available on the npm registry.

**Steps to reproduce:**

1.  Attempt to install `mcp-guide` globally using `npm install -g mcp-guide`.
2.  The installation fails with the following error message:

    ```
    npm error code ETARGET
    npm error notarget No matching version found for @modelcontextprotocol/sdk@^0.1.0.
    npm error notarget In most cases you or one of your dependencies are requesting
    npm error notarget a package version that doesn't exist.
    npm error A complete log of this run can be found in: /Users/richpav/.npm/_logs/2024-12-18T05_20_19_147Z-debug-0.log
    ```

3. Attempted to install with yarn using `yarn global add mcp-guide`

4. Yarn failed to install due to the same dependency issue, but provides the following prompt:
    ```
    Couldn't find any versions for "@modelcontextprotocol/sdk" that matches "^0.1.0"
    ? Please choose a version of "@modelcontextprotocol/sdk" from this list: (Use arrow keys)
    ❯ 1.0.4
      1.0.3
      1.0.2
      1.0.1
      1.0.0
      0.7.0
      0.6.1
      0.6.0
      0.5.0
      0.4.0
    ```
    No version was selected as they may not be compatible.

**Troubleshooting steps already taken:**

*   Updated npm to the latest version using `npm install -g npm@latest`.
*   Cleared the npm cache using `npm cache clean --force`.
*   Attempted installation using `yarn global add mcp-guide`
*   Confirmed the dependency issue with yarn, by observing the prompt that displayed the available versions.

**Additional information:**

*   Operating System: macOS 15.2 (24C101)
*   Node.js version: v23.4.0
*   npm version: 10.9.2
*   yarn version: 1.22.22

It appears that the `mcp-guide` project is requesting a specific version (`^0.1.0`) of `@modelcontextprotocol/sdk` which is not available. The available versions are higher.

Please let me know if you need any further information.

Thank you for your time.

**Comment by qpd-v** [comment]
on it

**Comment by qpd-v** [comment]
Fixed

**Comment by qpd-v** [comment]
This issue has been fixed in version 0.1.2 which was just published to npm. The changes include:

1. Updated the @modelcontextprotocol/sdk dependency from ^0.1.0 to ^1.0.4 to use the latest stable version
2. Fixed all version references in the codebase

You can now install the package using:
```bash
npm install -g mcp-guide
# or
yarn global add mcp-guide
